src/dotnet/CarbonAware.WebApi/api.yaml

```diff
@@ -0,0 +1,152 @@
+openapi: 3.0.0
+info:
+  description: 'Carbon Aware REST API'
+  version: 0.1.0
+  title: Carbon Aware REST API
+
+paths:
+  '/sci-scores':
```

Comment:
I'm confused here. Isn't this supposed to be CI centric? Why sci-score is involved now? Probably @esgraham knows more?

Reply:
The target use-case is reporting the marginal carbon intensity that will be used to calculate the sci-score in the future.  The base `/sci-scores` endpoint is still hardcoded to static data.  This pattern allows a user to calculate just one sci-score component without requiring the full calculation.

Reply:
I like the fact that the route is sci-scores/marginal-carbon-intensity.  I feel that it does provide flexibility for future sci components and tells the api user that the CI value is a part of a bigger calculation.